clang/lib/CIR/CodeGen/TargetInfo.cpp

```diff
@@ -305,6 +305,33 @@ class SPIRVTargetCIRGenInfo : public CommonSPIRTargetCIRGenInfo {
 
 } // namespace
 
+//===----------------------------------------------------------------------===//
+// NVPTX ABI Implementation
+//===----------------------------------------------------------------------===//
+
+namespace {
+
+class NVPTXABIInfo : public ABIInfo {
+  mlir::MLIRContext &mlCtx;
+
+public:
+  NVPTXABIInfo(CIRGenTypes &cgt, mlir::MLIRContext &mlCtx)
```

Comment:
Remove the `mlir::MLIRContext &mlCtx`, if we need in the future we could grab the context through `CIRGenTypes`.